### DISCUSSION TITLE
Import fail  `./infra/example-configuration.sh`

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#524] [General]
```
./infra/example-configuration.sh
./infra/example-configuration.sh: line 15: jq: command not found
POST /schema

##########################
#      IMPORT FAILED     #
##########################

```
Os- Windows 11 WSL2/Ubuntu

**Comment by johnaohara** [comment]
You need to install the jq (https://stedolan.github.io/jq/) utility

**Reply by shivarm** [reply]
installed but still fail

**Reply by shivarm** [reply]
I have tried lot but still got fail. No problem, will try again and again

**Reply by jesperpedersen** [reply]
Seems like a `%PATH%` issue

**Reply by shivarm** [reply]
What do you think @johnaohara

**Reply by jesperpedersen** [reply]
Did setting the correct path help ?

**Reply by shivarm** [reply]
No

**Reply by johnaohara** [reply]
There are a number of issues with running the script on windows, not having jq installed by default on the linux image, and the docker network interface not being available from inside the WSL2 env.

So, we either need to provide windows scripts, or replace the 'infra/` dir

**Reply by shivarm** [reply]
Now problem figureout for Windows, it's working fine but only convention is we need to keep runing docker in background to import example data. we can close this #537
